bescort.lic

```diff
@@ -144,6 +144,10 @@ class Bescort
       [
         { name: 'gondola', regex: /gondola/i, description: 'Gondola between Shard and Leth Deriel' },
         { name: 'mode', options: %w[north south], description: 'Which direction are you going?' }
+      ],
+      [
+        { name: 'desert', regex: /desert/i, description: 'Hidasharon Desert on Mriss for hunting armadillos' },
+        { name: 'mode', options: %w[adult juvenile elder oasis exit], description: 'Where to?'}
```

Comment:
`elder` and `oasis` are placeholders currently. I think they can be added at a later time (when someone's able to hunt there)